udata/templates/embed-dataset.html

```diff
@@ -5,7 +5,7 @@
     }
     .dataset-card {
         border: 1px solid #EEE;
-        min-width: 300px;
+        min-width: 0;
```

Comment:
@davidbgk no prob – I just added some screenshots to pinpoint where the problem was (look at the _tablet breakpoint_).